devtools/cli/src/main/java/io/quarkus/cli/Encryptor.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.cli;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.SecureRandom;
+import java.util.Base64;
+import java.util.concurrent.Callable;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+import io.quarkus.cli.common.OutputOptionMixin;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "encryptor", header = "Encrypt Secrets in AES/GCM/NoPadding")
+public class Encryptor implements Callable<Integer> {
+    @Option(required = true, names = { "-s", "--secret" }, description = "Secret")
+    String secret;
+
+    @Option(required = true, names = { "-k", "--encryption-key" }, description = "Encryption Key")
```

Comment:
I'd propose making it optional - creating a good encryption key will likely require users use some generation utility anyway, otherwise users can try some weak keys, which is in FileVaultUtils, if the secret is not provided, we generate a secure random one, use it, and print it alongside the encrypted value - I can add to this PR if you'd like or please copy the code :-)

Reply:
Similar approach is followed in Form and now OIDC if the required encryption keys are not manually configured

Reply:
Sure, I'll add that.